decidim-elections/app/permissions/decidim/elections/admin/permissions.rb

```diff
@@ -66,6 +68,10 @@ def allow_if_not_started
           toggle_allow(election && !election.started?)
         end
 
+        def allow_if_finished
+          toggle_allow(election && election.finished?)
+        end
```

Comment:
We're starting to have a lot of business logic in the permissions system. I think this logic belongs to the commands. The permission system should only check if the user can or can not do something.